Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -202,6 +202,97 @@ - (BOOL)containsPasteboardTypes:(NSArray*)pasteboardTypes {
 
 /**
  @Status Caveat
+ @Notes only support one item due to platform limit.
+ */
+- (NSArray<NSDictionary<NSString*, id>*>*)items {
+    WADDataPackageView* wasDataPackageView = _getClipboardContent();
+    id ret = [NSMutableDictionary dictionary];
+
+    if ([wasDataPackageView contains:[WADStandardDataFormats text]]) {
+        [ret setValue:[UIPasteboard _getStringFromDataPackageView:wasDataPackageView] forKey:(NSString*)kUTTypeText];
+    }
+
+    if ([wasDataPackageView contains:[WADStandardDataFormats bitmap]]) {
+        [ret setValue:[UIPasteboard _getImageFromDataPackageView:wasDataPackageView] forKey:(NSString*)kUTTypeImage];
+    }
+
+    if ([wasDataPackageView contains:[WADStandardDataFormats uri]]) {
+        [ret setValue:[UIPasteboard _getURLFromDataPackageView:wasDataPackageView] forKey:(NSString*)kUTTypeURL];
+    }
+
+    NSMutableArray<NSDictionary<NSString*, id>*>* items = [[[NSMutableArray alloc] init] autorelease];
```

Comment:
do you know if the reference platform returns nil for items if nothing is found, or if it returns an empty array as you're doing here?